examples/node/package.json

```
@@ -5,8 +5,9 @@
     "main": "app.js",
     "author": "",
     "license": "Apache 2.0",
+    "type": "module",
     "dependencies": {
         "cli-color": "^1.0.0",
-        "matrix-js-sdk": "^32.0.0"
+        "matrix-js-sdk": "^34.5.0-rc.0"
```

In any case, I've prevaricated long enough that we can use 34.5 final now:

```suggestion
        "matrix-js-sdk": "^34.5.0"
```